src/models/methods/baseMethod.ts

```diff
@@ -23,4 +23,5 @@ export interface BaseResponse {
     error?: string
     request?: Response
     api_version?: number
+    searched_all?: boolean // used in `tx`
```

Comment:
Why does this need to be in `baseResponse`? Why can't we just put it in `TxResponse`?

Reply:
whoops, u rite